### ISSUE TITLE
screenshot -raw should autodetect whether to use -doublesize option

### ISSUE THREAD

**Issue by MBilderbeek** [openMSX/openMSX#1062]
When using screenshot -raw to get (indeed) raw screenshots, the user now has to think about whether to use 320x240 of 640x480 screenshots. That's not very comfortable if you want to put this behind a hotkey. 

Why can't openMSX find out itself when to use -doublesize? (Use it automatically for screen TEXT80, 6 and 7.)

**Comment by MBilderbeek** [comment]
Or perhaps use something like this:
`bind PRINT {if {[lsearch {TEXT80 6 7} [get_screen_mode]] == -1} { screenshot -raw -guess-name} else { screenshot -raw -doublesize -guess-name}}`

**Comment by sdsnatcher** [comment]
Agreed. But it would be nice to be able to force the -doublesize if necessary.

**Comment by MBilderbeek** [comment]
You can always force it by using the -doublesize parameter, of course. But perhaps that's not what you meant?

**Comment by sdsnatcher** [comment]
I thought that once the autodetection would be implemented, the -doublesize could be gone. I wanted to make sure that it would stay.      :)

**Comment by MBilderbeek** [comment]
This solution only changes the hotkey. But we could also implement something similar for the Tools -> Capture -> screenshot menu item. Maybe add an 'auto' option, that does this and which is selected by default?
What do you think @m9710797 ?

**Comment by m9710797** [comment]
Good idea. I also think it should remain possible to override the auto detection, in *both* directions.
Maybe we can add an option like: `-size [auto|320|640]`
And the (old) `-doublesize` option then does the equivalent of `-size 640`.

For V9990 we could also support sizes `960` and `1280`. But let's leave that for a future extension (not bound to this ticket). It's also low priority IMHO.

One corner case is screen splits between modes of different widths. Actually now that I think of it (but without having looked at the code, I might be miss remembering), create a *raw* screenshot operates on `RawFrame` objects. These remember per line how wide it is (320, 640, ...). So we could calculate the maximum width of all lines. And that might actually be easier than figuring out the screen mode (because e.g. screen mode from which VDP?).

**Comment by MBilderbeek** [comment]
Yes, this is about raw mode only. So, what would it look like if we use that info instead?
